parent/pom.xml

```diff
@@ -399,6 +399,7 @@
     <jmh-version>1.21</jmh-version>
     <jodatime-bundle-version>1.6.2</jodatime-bundle-version>
     <jodatime2-bundle-version>2.10</jodatime2-bundle-version>
+    <jota-bundle-version>0.9.10</jota-bundle-version>
```

Comment:
Is Jota Java library a bundle? Not sure.

Reply:
No it's a lib sorry! It is a mistake